webrender/src/render_backend.rs

```diff
@@ -817,6 +817,16 @@ impl RenderBackend {
                         self.resource_cache.clear(mask);
                         return true;
                     }
+                    DebugCommand::SimulateLongSceneBuild(time_ms) => {
+                        self.scene_tx.send(SceneBuilderRequest::SimulateLongSceneBuild(time_ms)).unwrap();
+                        return true;
+                    }
+                    DebugCommand::SimulateLongLowPrioritySceneBuild(time_ms) => {
```

Comment:
should just be a bool parameter of `SimulateLongSceneBuild`

Reply:
I almost did that but eventually we'll have SimulateLongFrameBuilder and SimulateLongRender as well and then having the SB ones separate instead of a bool feels more natural (totally splitting hairs here).